tools/yoink/templates/documentation.md.j2

```diff
@@ -0,0 +1,24 @@
+<!-- THIS FILE IS AUTOGENERATED BY THE YOINK TOOL -->
+
+# Source {{source_name | title}}
+
+{% if manifest.get('description') -%}
+## Description
+{{ manifest.get('description') }}
+{% endif -%}
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+{% for property, details in manifest.get('spec').get('connection_specification').get('properties').items() -%}
+| `{{ property }}` | `{{ details.type }}` | {{details.get('title')}}. {{ details.get('description', '') | replace('\n', ' ') }} | {% if 'default' in details %}{{ details.default }}{% endif %} |
```

Comment:
💅 Im a fan of moving the if/else/default logic to the view controller. Just for highlighting, and testing.